surface_logs_create_schema_public_if_not_exists.txt

```diff
@@ -0,0 +1,123 @@
+The files belonging to this database system will be owned by user "postgres".
```

Comment:
dumps collected by
```
docker-compose up surface
docker logs surface > filename
```

with `CREATE SCHEMA IF NOT EXISTS public;` in surface_dump.sql